back/.env

```diff
@@ -1,2 +1,5 @@
 NODE_ENV=development
 PORT=8081
+API_MOCK=false
+MONGO_URL=mongodb://localhost:27017/planning-poker
+API_URL=/api
```

Comment:
Not sure if we should name this file .env-sample and ignore .env (we should add instructions to the readme.md about how to configure this), another option could be playing in the CD removing this file (works by default).

Reply:
@Nasdan could shed some light about the approach, my concern is having .env in production as well

Reply:
Yes you are right in my opinion, I prefer also do something .env-dev, or .env-sample, for me is perefct

Reply:
We should remove this file. [Reference](https://github.com/motdotla/dotenv#should-i-commit-my-env-file)

In other projects we name it like `.env.example`, check for example: https://github.com/Lemoncode/master-frontend-lemoncode/tree/master/07-cloud/03-docker/05-heroku-back/01-implementation/back